README.md

```diff
@@ -1,5 +1,69 @@
 # Learning analytics playground
 
+## Description
+
+This tool is a first draft of connecting an LMS interface to a Graylog server to create a dataset 
+of learning logs.
+
+It is intended to be used locally. 
+Currently, it is using LMS from OpenEdx and the FUN's release of dogwood.
```

Comment:
Maybe add a link to openfun/openedx-docker?

```suggestion
Currently, it is using OpenEdx's LMS service (with the fun-flavored dogwood's release).
```

Reply:
Good idea